_summaries/dosovitskiy2020an_1.md

```diff
@@ -1,19 +1,43 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: dosovitskiy2020an
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: elampietti
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Traditionally, computer vision problems were attempted with CNN architectures, however, due to the success of Transformers with NLP tasks, this paper experiments with applying Transformers to computer vision problems.
+
+This is done by dividing the images into patches which are then flattened and transformed into a sequence of linear embeddings.
+Positional information is retained by adding positional embeddings to the patch embeddings.
+This sequence of embeddings is then fed into a standard Transformer. 
+The model is then trained with supervision for image classification.
+An overview of this Vision Transformer model is picture below in Figure 1.
```

Comment:
This is a good, brief, and easy to understand summary of the approach and architecture. I would say maybe just include a brief comparison to previous, CNN based approaches as well as similar approaches like igpt.